build.sc

```diff
@@ -89,13 +81,13 @@ trait Bot4sTelegramModule extends CrossScalaModule {
     library.catsCore,
     library.catsFree,
     library.sttpCore,
-    library.slogging
+    library.scalaLogging
```

Comment:
Add 
```
library.logback
```

Here, that should fix the issue you raised in https://github.com/bot4s/telegram/pull/94#issuecomment-664032754